src/index.html

```diff
@@ -12,137 +12,285 @@
+      <img class="tick" src="imgs/check.svg" alt="" />
+      <h3>Foster a deeper bond</h3>
+      <p class="text-align-center">Teach your parrots tricks and behaviors through in-cage training.</p>
+    </div>
+  </div>
+</section>
 
-    <div class="flex-container flex-container--row-gap">
-      <div class="flex-container">
-        <img class="icons icons--on-dark" src="imgs/icon-enrichment.svg" alt="" />
-        <h3>Keep your flock safe</h3>
-        <p>
-          We care about the well-being of your flock as much as you do. That is why we are constantly working on improving the safety and quality of BeakBox. While making design iterations, we listen
-          to the feedback of animal welfare experts, local vets, and parrot owners to ensure that it is 100% safe for your parrots.
-        </p>
+<section>
+  <div class="main-container main-container--narrow banner light-background">
+    <div class="flex-container money-back">
+      <h2>Hassle-free <span style="color: #ff662b">30-day money-back</span> guarantee</h2>
+      <div class="flex">
+        <a class="button" href="https://shop.mybeakbox.com">Give BeakBox a go</a>
       </div>
+      <img src="imgs/yellow-bird.png" alt="" />
+    </div>
+  </div>
+</section>
 
-      <div class="flex-container">
-        <img class="icons icons--on-dark" src="/imgs/icon-learning-tool.svg" alt="" />
 
-        <h3>Enrich your parrot's life</h3>
-        <p>
-          With BeakBox, your pet bird can enjoy hours of fun and mental stimulation, promoting their overall well-being and reducing the risk of boredom-related behaviors. The interactive and engaging
-          nature of the toy encourages your parrots to explore and learn, helping to keep their minds active and alert.
-        </p>
-      </div>
 
-      <div class="flex-container">
-        <img class="icons icons--on-dark" src="/imgs/icon-music-anytime.svg" alt="" />
-        <h3>Transform your parrot's learning</h3>
-        <p>
-          BeakBox is more than just a toy for entertainment; it is also a perfect learning tool for your pet parrots. With BeakBox, you can train your parrots to unlearn destructive behaviours and
-          learn new tricks, all while stimulating their minds and bonding with your featheres friends.
-        </p>
-      </div>
+<section>
+  <div class="main-container main-container--narrow flex-container flex-container--center-content">
+    <div class="flex-container">
+      <h2><span style="color: #ff662b">BeakBox </span> vs. other bird toys</h2>
+      <p>BeakBox isn't just a toy; it's your parrot's second-best friend, right after you. It challenges your parrot's
+        mind and keeps them engaged all day.</p>
     </div>
+    <table class="comparison">
+      <tr>
+        <th></th>
+        <th><span style="color: #ff662b">BeakBox</span></th>
+        <th>Other bird toys</th>
+      </tr>
+      <tr>
+        <td>100% safe materials</td>
+        <td><img src="imgs/check-mark.svg" alt=""></td>
```

Comment:
These all need `alt` attributes with text otherwise screenreaders will ignore them and the table will be meaningless.